sql/moz-fx-data-shared-prod/telemetry_derived/scalar_percentiles_v1/metadata.yaml

```
@@ -0,0 +1,6 @@
+friendly_name: |-
+  Telemetry Derived - Scalar Percentiles
+description: |-
+  [DESCRIPTION_MISSING]
+owners:
+- nobody@mozilla.com
```

Alternatively, instead of using nobody, we could also say it's owned by the data infra WG

```suggestion
- data-platform-infra-wg@mozilla.com
```

---

We can do that, I'll make the change now.